### ISSUE TITLE
Not able to install rancher-backup chart

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#32152]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

- Install rancher-backup 1.0.400-rc3 version
- Error seen : `This chart requires rancher-backup-crd=match but no matching chart was found`

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): On 2.5-head commit id: `4e70ed2 `
- Installation option (single install/HA): HA

**Comment by vincent99** [comment]
The annotation says the versions have to match but there is no crd chart published for `1.0.400-rc03`, only `…rc02`.

```
CRD: (6) [                "1.0.400-rc02", "1.0.301-rc01", "1.0.301-rc00", "1.0.300", "1.0.201", "1.0.200"]
App: (7) ["1.0.400-rc03", "1.0.400-rc02", "1.0.301-rc01", "1.0.301-rc00", "1.0.300", "1.0.201", "1.0.200"]
```

**Comment by sowmyav27** [comment]
**on 2.5-head commit id: `31f00c1 ` verified using backup restore chart - `rancher-backup:1.0.400-rc03`**

- Deployed rancher HA
- Deployed a GKEv2 cluster and an AWS node drive rke cluster
- deployed rancher-backup chart. No issues seen
- Take backup b1
- b1 was taken successfully.
